redisvl/utils/threshold_optimizer/schema.py

```diff
@@ -0,0 +1,11 @@
+from typing import List, Optional
+
+from pydantic import BaseModel, Field
+from ulid import ULID
+
+
+class TestData(BaseModel):
+    q_id: str = Field(default_factory=lambda: str(ULID()))
```

Comment:
`_id` feels maybe cleaner than knowing what the `q` is for ;)

Reply:
used q_id since that's ranx convention for this but yeah _id works